redaxo/src/core/lib/packages/addons/addon.php

```diff
@@ -43,6 +43,24 @@ public static function get($addon)
         return self::$addons[$addon];
     }
 
+    /**
+     * Returns the addon by the given name.
+     *
+     * @param string $addon Name of the addon
+     *
+     * @throws RuntimeException if the addon does not exist
+     *
+     * @return self
+     */
+    public static function require(string $addon): rex_package
+    {
+        if (!isset(self::$addons[$addon])) {
+            throw new RuntimeException(sprintf('Required addon "%s" does not exist.', $addon));
```

Comment:
in den anderen methoden haben wir eine `InvalidArgumentException`

Reply:
Laut PHP-Doku:

> Exception thrown if an argument is not of the expected type.

https://www.php.net/manual/de/class.invalidargumentexception.php

Daher passt es für mich hier nicht.